CHANGELOG.md

```diff
@@ -18,6 +18,8 @@
 - [#2338](https://github.com/influxdb/influxdb/pull/2338): Fix panic if tag key isn't double quoted when it should have been
 - [#2340](https://github.com/influxdb/influxdb/pull/2340): Fix SHOW DIAGNOSTICS panic if any shard was non-local.
 - [#2351](https://github.com/influxdb/influxdb/pull/2351): Fix data race by rlocking shard during diagnostics.
+- [#2350](https://github.com/influxdb/influxdb/pull/2350): Issue fix for :influxd -hostname localhost.
```

Comment:
OK. I thought Jason's change was a once-off, and the format was more like:

#1234 <URL> description.....fixes 4678, 3789, 23789

where 1234 is a PR, not an issue.

I think the link is wrong though in your change. It works, but it's actually an issue.